modules/gapi/include/opencv2/gapi/garg.hpp

```diff
@@ -235,8 +235,50 @@ inline GRunArgsP& operator += (GRunArgsP &lhs, const GRunArgsP &rhs)
 
 namespace gapi
 {
+    /**
+     * \addtogroup gapi_serialization
+     * @{
+     *
+     * @brief G-API functions and classes for serialization and deserialization.
+     */
+    /** @brief Binds deserialized output GRunArgs to GRunArgsP which can be used by GCompiled.
+     *
+     * Example:
+     * @code{.cpp}
+     * cv::GCompiled compd;
+     * auto graph = cv::gapi::deserialize<cv::GComputation>(bytes);
+     * this_graph = cv::util::make_optional(graph);
+     *
+     * compd = this_graph->compile();
+     * auto in_args  = cv::gapi::deserialize<cv::GRunArgs>(bytes_in);
+     * auto out_args = cv::gapi::deserialize<cv::GRunArgs>(bytes_out);
+     * this_compd(std::move(in_args), cv::gapi::bind(out_args));
```

Comment:
Please move this code to a real `.cpp` file in the `modules/gapi/samples` directory and add here via `@snippet` macro. Grep for `@snippet` in our public headers to find the examples.

Why - this way the code will be compiled as part of OpenCV and will remain valid - in case if the API changes then.

Reply:
Done